### PR TITLE
28 profile component

### PR DESCRIPTION
**Pull request by AbdallahAmmar96**
this component to render the personal information for the student 
and make a new common component name TextCard and make the style for it 
the TextCard render a title and the data for this title 
for example : 
### important note 
- the data below do not need a static file because of this data  will come from the fetch 
and will be stored in the state object 

Relates #28

### CODE REVIEW

File: Client/src/components/contanier/student/Profile/index.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+import TextCard from '../../../common/TextCard'
+import './index.css';
 
 export default class Profile extends Component {
+  state = {
+    level: 6,
+  }
+
   render() {
+    const label = ['Name :', 'Phone Number :', 'Amideast ID :']
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Profile/index.js]: I think that if you put this in the state, you can simplify your work when you fetch the data !!

File: Client/src/components/contanier/student/Profile/index.js [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
+    const data = ['Abdallah Amamr', '0597185554', '59763']
     return (
-      <div>
-        from Profile
+      <div className='profile'>
+        <h1 className='profile-titel'>Profile</h1>
+        <h2 className='profile-level' >Level {this.state.level}</h2>
+        {label.map((item, index) => {
+          return (
+            <TextCard data={data[index]} label={item} key={index} />
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Profile/index.js:20]: ### Look at the annoying difference in font size
React App 4-22-2019 4-03-53 PM

**Review by AsalaKM — COMMENTED** [review]
good job

File: Client/src/components/contanier/student/Profile/index.js (hunk 2 above)
Comment (AbdallahAmmar96) [reply Client/src/components/contanier/student/Profile/index.js:20]: I make some edits you can see it 
and hope you can give me some suggestions

File: Client/src/components/contanier/student/Profile/index.js (hunk 1 above)
Comment (AbdallahAmmar96) [reply Client/src/components/contanier/student/Profile/index.js]: Done

File: Client/src/components/contanier/student/Profile/index.js (hunk 2 above)
Comment (AsalaKM) [reply Client/src/components/contanier/student/Profile/index.js:20]: React App 4-22-2019 5-26-59 PM


## You Can :
1- Change the font family.
1-  Put the label and data in the same line in all screens

**Review by AsalaKM — APPROVED** [review]

File: Client/src/components/contanier/student/Profile/index.js (hunk 2 above)
Comment (AbdallahAmmar96) [reply Client/src/components/contanier/student/Profile/index.js:20]: I changed the font family

**Comment by AsalaKM** [comment]
@AbdallahAmmar96 
Don't merge any PR before take a review from all the team member

**Pull request merged**